src/components/Navigation.js

```diff
@@ -29,8 +29,8 @@ export const navigationCSS = ({ interactionIsIdle }: NavState) => ({
 });
 
 export const Navigation = (props: NavProps) => {
-  const { children, getStyles, isFullscreen, isModal } = props;
-  return !isTouch() ? (
+  const { children, getStyles, isFullscreen, isModal, hideNavigationOnTouchDevice } = props;
```

Comment:
I thought about the `showNavigationButtonsOnTouchDevice ` as well. But because you already had `hideControlsWhenIdle`, i went with the one i had, Anyways, I update the code! Thanks for the feedback.

Reply:
That is 100-percent understandable and I definitely would have done the same! I'm just thinking about the ease of implementation but code style is also important.